modules/Planner/planner_addProcess.php

```diff
@@ -295,6 +295,22 @@
                         header("Location: {$URL}");
                     }
                 }
+
+                //Notify participants
+                //GET CHECK BOX VALUE
+                if (isset($_POST['notify'])) {
+                    //Create notification for all people in class except me
+                    $dataClassGroup = array('gibbonCourseClassID' => $gibbonCourseClassID);
+                    $sqlClassGroup = "SELECT * FROM gibbonCourseClassPerson INNER JOIN gibbonPerson ON gibbonCourseClassPerson.gibbonPersonID=gibbonPerson.gibbonPersonID WHERE gibbonCourseClassID=:gibbonCourseClassID AND status='Full' AND (dateStart IS NULL OR dateStart<='".date('Y-m-d')."') AND (dateEnd IS NULL  OR dateEnd>='".date('Y-m-d')."') AND (NOT role='Student - Left') AND (NOT role='Teacher - Left') ORDER BY role DESC, surname, preferredName";
+                    $resultClassGroup = $connection2->prepare($sqlClassGroup);
+                    $resultClassGroup->execute($dataClassGroup);
+                    while ($rowClassGroup = $resultClassGroup->fetch()) {
+                        if ($rowClassGroup['gibbonPersonID'] != $_SESSION[$guid]['gibbonPersonID']) {
+                            $notificationText = (__("Lesson plan “".$name."” has been created."));
```

Comment:
Strings inside __() are pulled out of the code so that they can be shared and translated, so this is one case where they can't be broken up and concatenated the way we would a normal string. The way around this is to use `sprintf()` with a `%1$s` token to represent the variable you need to add to the string. In this case, formatting it would look similar to: `sprintf(__('Lesson plan %1$s has been created.'), $name);`

I do wish all this stuff was in the docs for you! but for now the best way is to learn-by-doing and we're happy to help with suggestions 😄